app/routing/ArrivalsRouter.scala

```diff
@@ -42,4 +44,10 @@ class ArrivalsRouter @Inject() (
       case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE) => v2Arrivals.createArrivalNotification()
       case _                                                    => v1Arrivals.createArrivalNotification()
     }
+
+  def getArrivalsForEori(updatedSince: Option[OffsetDateTime] = None): Action[Source[ByteString, _]] = route {
```

Comment:
Can not see any usage of updatedSince optional query param in code also not used in transit-movements backend connector api. any specific reason to use this optional query param?